src/components/EasyDropdown/EasyDropdown.test.js

```diff
@@ -4,7 +4,8 @@ import { act } from 'react-dom/test-utils';
 
 import Button from '../Button';
 
-import { Menu } from '../Dropdown';
+import Dropdown from '../Dropdown';
```

Comment:
Why the new import without corresponding new usage?

Reply:
Guess this should be a change request, but it's minor, so 👍

Reply:
Updated! 👍